Cash register

ijassiem opened the pull request.

ijassiem committed:
Added mod,divide,multiply

ijassiem committed:
Added functions and unit tests DGDT-14

tyronevb reviewed (commented):
Have a look at the proposed structure for commit messages. 

Out of interest, how are you entering your commit messages? Via `git commit -m "commit message"`? Or are you doing `git commit` which then launches a text editor for you to enter your commit message?

james-smith-za reviewed (commented):
Put a file called `.gitignore` in the root of your git repo. There are quite a few files in your git repo that don't need to be here.

```
*.egg-info
__py_cache__
```

james-smith-za commented on cash_register/cash_register.py:
```diff
@@ -0,0 +1,39 @@
+
+def vat(value):
+    """Calculate VAT."""
+    return round(value * 0.14, 2)
```
VAT is now 15% ;-)

ijassiem replied:
Fixed

james-smith-za commented on multiply.py:
```diff
@@ -0,0 +1,11 @@
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```
Make sure you update your docstrings. This will satisfy pydocstyle, but it's not terribly useful.

james-smith-za replied:
Same for mod.py and divide.py

ijassiem replied:
These docstrings have been corrected in the functions branch.

james-smith-za commented on testing/test_cash_register.py:
```diff
@@ -0,0 +1,47 @@
+import cash_register.cash_register as cr
+import unittest
+
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```
Same feedback here.

ijassiem replied:
Fixed

ijassiem committed:
Implement changes relating to comments in pull request

These changes pertains to the comments indicated in the pull request: "Cash register #2"

Added a .gitignore file and corrected docstrings in cash_register.py and test_cash_register.py.

Resolves: "DGDT-14"
